contracts/chain-adapters/Optimism_Adapter.sol

```diff
@@ -72,8 +72,18 @@ contract Optimism_Adapter is CrossDomainEnabled, AdapterInterface {
             l1Weth.withdraw(amount);
             l1StandardBridge.depositETHTo{ value: amount }(to, l2GasLimit, "");
         } else {
-            IERC20(l1Token).safeIncreaseAllowance(address(l1StandardBridge), amount);
-            l1StandardBridge.depositERC20To(l1Token, l2Token, to, amount, l2GasLimit, "");
+            IL1StandardBridge _l1StandardBridge = l1StandardBridge;
+
+            // Check if the L1 token requires a custom bridge. If so, use that bridge over the standard bridge.
```

Comment:
Only nit is I think it'd be helpful to link to some source where we can look up custom bridges per L1 token. otherwise LGMT